base/util.jl

```diff
@@ -445,9 +445,21 @@ function securezero! end
     ccall(:memset, Ptr{T}, (Ptr{T}, Cint, Csize_t), p, 0, len*sizeof(T))
 unsafe_securezero!(p::Ptr{Cvoid}, len::Integer=1) = Ptr{Cvoid}(unsafe_securezero!(Ptr{UInt8}(p), len))
 
+"""
+    Base.getpass(message::AbstractString) -> `Base.SecretBuffer`
+
+Display a message and wait for the user to input a secret, returning an `IO`
+object containing the secret.
+
+Note that on Windows, the secret might be displayed as it is typed; see
+`Base.winprompt` for securely retrieving username/password pairs from a secure
+graphical interface.
+"""
+function getpass end
+
 if Sys.iswindows()
 function getpass(prompt::AbstractString)
-    print(prompt)
+    print(prompt, ':')
```

Comment:
`": "` ?

Reply:
The extra space there doesn't matter all that much because (ideally) the typed input doesn't get echo'ed back.  In any case, this was required to match `Base.prompt`'s behavior.

Reply:
The system might still show a blinking cursor of some kind.  On MacOS it displays a key cursor
![image](https://user-images.githubusercontent.com/2913679/41734410-d4d87cae-7554-11e8-9579-e3bdaf865a69.png)
I think the `Base.prompt` should also show a space.